src/lib/rest/StringFilter.cpp

```diff
@@ -1587,13 +1662,29 @@ bool StringFilter::qMatch(ContextElementResponse* cerP)
     {
       ContextAttribute* caP = cerP->contextElement.getAttribute(itemP->attributeName);
 
-      if ((itemP->op == SfopExists) && (caP == NULL))
```

Comment:
Just thinking if the reviewer will get crazy trying to find changes related with the new functionality in the new PR that are not actually in that PR... 

Although from a pragmatic point of view, I wouldn't try to remove this bits from this PR to be  "moved" to the next. That could be complex and error prone. Thus, I agree in NTC (but try to do the cut in a cleaner way next time ;)